tests/fixtures/renderer_test_user_rules/test_rule_1.yaml

```diff
@@ -0,0 +1,22 @@
+es_host: 127.0.0.1
+es_port: 80
+index: test_index
+type: any
+alert:
+  - email:
+      email: example@gmail.com
+      email_config_name: mail_1
+  - email:
+      email: test@gmail.com
+      email_config_name: mail_2
+  - email:
+      email: test2@gmail.com
+      email_config_name: default
+  - slack:
+      slack_username_override: test_user
+      slack_channel_override: test_channel
+      slack_config_name: slack_1
```

Comment:
Let's rename `<notifier>_config_name` to `<notifier>_id`

Reply:
Changed